.yarn/sdks/eslint/bin/eslint.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const {existsSync} = require(`fs`);
+const {createRequire, createRequireFromPath} = require(`module`);
+const {resolve} = require(`path`);
+
+const relPnpApiPath = "../../../../.pnp.cjs";
+
+const absPnpApiPath = resolve(__dirname, relPnpApiPath);
+const absRequire = (createRequire || createRequireFromPath)(absPnpApiPath);
+
+if (existsSync(absPnpApiPath)) {
+  if (!process.versions.pnp) {
+    // Setup the environment to be able to require eslint/bin/eslint.js
+    require(absPnpApiPath).setup();
+  }
+}
```

Comment:
Consider

```suggestion
if (existsSync(absPnpApiPath) && !process.versions.pnp) {
  // Setup the environment to be able to require eslint/bin/eslint.js
  require(absPnpApiPath).setup();
}
```

Similarly in the next file

Reply:
These files are generated, unless I'm mistaken.

Reply:
Yes, they were generated by yarn.